src/content/core/remote-execution.mdx

```diff
@@ -69,9 +69,44 @@ When running on remote, Okteto will automatically synchronize your local folder
 
 ## Ignoring files
 
-Okteto looks for a file named `.oktetoignore` in your local folder. If this file exists, the CLI will exclude any files and directories that match patterns in it.
-This helps to avoid unnecessarily synchronizing large or sensitive files and directories that are not used by the commands defined in the `deploy` section.
-`.oktetoignore` uses the same format as [.dockerignore files](https://docs.docker.com/engine/reference/builder/#dockerignore-file).
+Okteto looks for a file named `.oktetoignore` in your local folder. If this file exists, the CLI will exclude any files and directories in the command group that match patterns in it.
+This helps to avoid unnecessarily synchronizing large or sensitive files and directories that are not used by the commands defined in the `deploy`, `destroy` or `test` section.
+
+An example `.oktetoignore` file would be the following:
+
+```
+# applied to all commands
+docs
+
+# applied to deploy commands only
+[deploy]
+integration/*
+
+# applied to destroy commands only
+[destroy]
+app
+
+# applied to all test commands
+[test]
+app
+
+# applied to unit test command only
+[test.unit]
+integration
+```
+
+This `.oktetoignore` expands to the following rules:
+
+- For deploys, ignores `docs` and `integration/**`.
+- For destroys, ignores `docs` and `app`. 
+- For any test, ignores `docs` and `app`. 
+- For the unit test specifically, ignores `docs`, `app` and `integration`.
+
+Each command group in `.oktetoignore` uses the same format as [.dockerignore files](https://docs.docker.com/engine/reference/builder/#dockerignore-file).
+
+:::note
+For remote deploy and destroy operation okteto also reads the legacy `.oktetodeployignore` file but it will be deprecated in future versions.
```

Comment:
```suggestion
For remote `deploy` and `destroy` operations, Okteto also reads the legacy `.oktetodeployignore` file but it will be deprecated in future versions.
```